code/game/objects/items/debug_items.dm

```diff
@@ -333,3 +340,56 @@
 	if(!HAS_TRAIT(user, TRAIT_SECURITY_HUD))
 		hud = GLOB.huds[DATA_HUD_SECURITY_ADVANCED]
 		hud.remove_hud_from(user)
+
+// kinda works like hilbert, but not really
+/obj/item/map_template_diver
+	name = "Pseudo-world diver"
+	desc = "A globe that you can dive into a pseudo-world, but there's no way back."
+	icon_state = "hilbertshotel"
+	w_class = WEIGHT_CLASS_TINY
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+
+	var/datum/map_template/map_template = /datum/map_template/debug_target
+	var/working
+	var/turf_to_dive
+
+	var/live_server_warning
+
+/datum/map_template/debug_target
+	name = "Debug map to test"
+	mappath = '_maps/map_files/debug/multidir_sprite_debug.dmm'
+
+// friendly warning setter
+/obj/item/map_template_diver/Initialize()
+	. = ..()
+#ifndef LOWMEMORYMODE
+	live_server_warning = TRUE
+#endif
+
+/obj/item/map_template_diver/attack_self(mob/user)
+	. = ..()
+
+	if(live_server_warning)
```

Comment:
I want this blocked from people that don't have perms to prevent it being used as a gift from presents